docs/DeveloperGuide.adoc

```diff
@@ -410,40 +410,99 @@ Use case ends.
 Use case resumes at step 2.
 
 [discrete]
-=== Use case: Change appointment date (UC3)
+=== Use case: Add new appointment for a patient (UC3)
+Pre-condition: Patient exists in the system
 
 *MSS*
 
-1.  Patient calls to change appointment date
-2.  User checks whether the current appointment can be postponed.
+1.  Patient wants to have a new appointment
+2.  User wants to add this appointments for patient
+3.  ClerkPro adds this appointment
+
+Use case ends.
+
+*Extensions*
+
+* 2a. Actor inputs invalid format
++
+
+** 2a1. ClerkPro requests for correct input format +
+
+** 2a2. Actor inputs correct format +
++
+
+Steps 2a1-2a2 are repeated until the appointment has the correct format +
+
+Use case resumes at step 2
+
+* 2b. Actor inputs a appointment which is conflicted with other appointments
++
+
+** 2b1. Actor ask patient to make provide a new appointment date  +
+
+** 2b2. patient give a new appointment date +
++
+Steps 2b1-2b2 are repeated until the appointment is not conflicted
+
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Change appointment date (UC4)
+Pre-condition: Patient’s appointment exists
+
+*MSS*
+
+1.  Patient wants to Change a current appointment date
```

Comment:
I believe your MSS should begin at the user's interaction with the system.

Reply:
okay